clusterdock/cli.py

```diff
@@ -277,3 +268,15 @@ def _add_help(parser):
                         action='help',
                         default=argparse.SUPPRESS,
                         help='show this help message and exit')
+
+
+def _handle_etc_localtime(clusterdock_config_directory):
+    # As a workaround for https://github.com/docker/for-mac/issues/2396, we write the /etc/localtime
+    # into the clusterdock config directory, which can be mounted into containers.
+    clusterdock_config_directory_path = os.path.realpath(os.path.expanduser(clusterdock_config_directory))
+    logger.info('clusterdock_config_directory_path = %s', clusterdock_config_directory_path)
+    if not os.path.exists(clusterdock_config_directory_path):
```

Comment:
maybe wanna keep this creation outside, as its not to do with localtime thing but generally need this dir